.github/actions/createOrUpdateStagingDeploy/index.js

```diff
@@ -195,14 +195,31 @@ const {execSync} = __nccwpck_require__(3129);
  * @returns {Array}
  */
 function getPullRequestsMergedBetween(fromRef, toRef) {
-    const command = `git log --format="%s" ${fromRef}...${toRef}`;
+    const command = `git log --format="{[%B]}" ${fromRef}...${toRef}`;
     console.log('Getting pull requests merged between the following refs:', fromRef, toRef);
     console.log('Running command: ', command);
     const localGitLogs = execSync(command).toString();
-    return _.map(
-        [...localGitLogs.matchAll(/Merge pull request #(\d{1,6}) from (?!Expensify\/(?:master|main|version-))/g)],
+
+    // Parse the git log into an array of commit messages between the two refs
+    const commitMessages = _.map(
+        [...localGitLogs.matchAll(/{\[([\s\S]*?)\]}/gm)],
```

Comment:
So I have used your advice and split it up to two regexps. Here, I get the individual commit messages which are all inside of the `{[]}`